### PR TITLE
Feature/devs bellacos/design patterns

### PR DESCRIPTION
**Pull request by WilliamrDiaz**
Alejandro Fernandez
Jhossef Constain
William Diaz 
Guillermo Peña 
Julian Piamba

### CODE REVIEW

File: .vs/slnx.sqlite
Comment (zahydo) [inline .vs/slnx.sqlite:1]: Remove this

File: .vs/slnx.sqlite-journal
Comment (zahydo) [inline .vs/slnx.sqlite-journal:1]: Remove this

File: .vs/slnx.sqlite
Comment (WilliamrDiaz) [reply .vs/slnx.sqlite:1]: Done!

File: .vs/slnx.sqlite-journal
Comment (WilliamrDiaz) [reply .vs/slnx.sqlite-journal:1]: Done!

**Comment by zahydo** [comment]
Checked

**Pull request closed without merging**